.github/workflows/qemu-image-build.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           make init
           GIT_SHA=${{github.sha}}
           export PACKER_LOG=1
-          packer build -var "git_sha=${GIT_SHA}" -var-file="common-nix.vars.pkr.hcl" qemu-arm64-nix.pkr.hcl
+          sudo nix run github:supabase/postgres/${GIT_SHA}#packer -- build -var "git_sha=${GIT_SHA}" -var-file="common-nix.vars.pkr.hcl" qemu-arm64-nix.pkr.hcl
```

Comment:
Do we need sudo here ?

Reply:
We actually should not. On this one, I will revert to just using the packer that is baked in, and update it with the team working on qemu build when they are ready